### PR TITLE
Meta: Automatically generate release again

### PR DESCRIPTION
**Pull request by fregante**
For https://github.com/refined-github/refined-github/issues/4008#issuecomment-942723376

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -30,6 +30,15 @@ jobs:
       - uses: fregante/daily-version-action@v1
         name: Create tag if necessary
         id: daily-version
+      - name: Create release
```
Comment (FloEdelmann) [inline .github/workflows/release.yml:33]:
```suggestion
      - if: steps.daily-version.outputs.created
        name: Create release
```

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -30,6 +30,15 @@ jobs:
...
+        uses: actions/github-script@v4.0.2
```
Comment (FloEdelmann) [inline .github/workflows/release.yml:34]: Let's use the newest version directly.

```suggestion
        uses: actions/github-script@v5.0.0
```

**Review by FloEdelmann — CHANGES_REQUESTED** [review]

**Comment by fregante** [comment]
This did not work: https://github.com/refined-github/refined-github/actions/runs/1607743677

**Comment by FloEdelmann** [comment]
It's definitely the latter:

grafik

**Comment by FloEdelmann** [comment]
Ahh, of course `steps.version.outputs.version` is empty, if the step is called `daily-version`...

**Pull request merged**